Monika After Story/game/script-topics.rpy

```diff
@@ -3807,7 +3807,50 @@ label monika_morning:
         m "Well, maybe not as funny as me~" #Expand more maybe?
     return
 
-#Add one for the afternoon?
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_afternoon",category=['misc'],prompt="Good afternoon",pool=True))
+
+label monika_afternoon:
+    $ current_time = datetime.datetime.now().time().hour
+    $ sunrise_hour = int(persistent._mas_sunrise / 60)
+    $ sunset_hour = int(persistent._mas_sunset / 60)
+    if 4 <= current_time < 12:
```

Comment:
`content` now has the suntime helper functions (#2217), please merge in the new changes and use those.